test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlServerFhirStorageTestHelper.cs

```diff
@@ -185,8 +186,9 @@ SELECT t.name
         private SchemaInitializer CreateSchemaInitializer(string testConnectionString)
         {
             var config = new SqlServerDataStoreConfiguration { ConnectionString = testConnectionString, Initialize = true };
-            var schemaUpgradeRunner = new SchemaUpgradeRunner(config, NullLogger<SchemaUpgradeRunner>.Instance);
-            var schemaInformation = new SchemaInformation();
+            var schemaInformation = new SchemaInformation((int)SchemaVersion.V1, (int)SchemaVersion.V2);
```

Comment:
Is there a way we could determine max version automatically? Perhaps an extension method that does something like `Enum.GetValues(typeof(SchemaVersion)).Cast< SchemaVersion >().Max()`?

Reply:
We could technically do it that way. I was thinking it may make more sense to add a constants value like something below:

```dotnet
namespace Microsoft.Health.Fhir.SqlServer.Features.Schema
{
    public static class SchemaVersionConstants
    {
        public const int Max = (int)SchemaVersion.V2;
        public const int Min = (int)SchemaVersion.V1;
    }
}
```

Reply:
Yep also makes sense